send/scim_atlassian_mu.pm

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+SERVICE_NAME="scim_atlassian_mu"
+EXECSCRIPT="./scim-atlassian-connector.pl"
+
+FACILITY_NAME=$1
+DESTINATION=$2
+DESTINATION_TYPE=$3
+
+	#Test if destination is in not empty and in the correct format
```

Comment:
The indentation in this script is not right.

Reply:
thanks, fixed